inst/include/sampler.h

```diff
@@ -151,6 +151,16 @@ namespace smc {
         long GetNumber(void) const {return N;}
         ///Returns the number of evolution times stored in the history.
         long GetHistoryLength(void) const {return History.size();}
+        ///Returns the current particle set stored in the history.
+        population<Space> GetHistoryPopulation(long n) const {return History[n].GetValues();}
+        ///Returns a reference to the particle set stored in the history.
+        population<Space> & GetHistoryPopulationRefs(long n) {return History[n].GetValues();}
+        ///Returns the history flags
+        historyflags GetHistoryFlags(long n) {return History[n].GetFlags();}
```

Comment:
Is there a reason for this not to be declared `const`?

Reply:
I usually find strange to declare the references as const but in this particular situation i don not think that it will be a problem to declare them as const. I come from a C programming background so i am just beginning to grasp the subtleties of the language (i am working in cpp just for an year or so). I will remake it.